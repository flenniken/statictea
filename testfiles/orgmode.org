# $ # Testing the org mode prefix.

# $ nextline name = "org mode"
hello {name}

# Normal org mode comment.
