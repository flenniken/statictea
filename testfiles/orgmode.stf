id stf file version 0.0.0

## Test the org mode prefix (# $).

--- file cmd.sh command
$statictea -p="# $" -t=orgmode.org >stdout 2>stderr
--- endfile

--- file orgmode.org
# $ # Testing the org mode prefix.

# $ nextline name = "org mode"
hello {name}

# Normal org mode comment.
--- endfile

--- file stdout.expected

hello org mode

# Normal org mode comment.
--- endfile

--- file stderr.expected
--- endfile

--- expected stdout.expected == stdout
--- expected stderr.expected == stderr

