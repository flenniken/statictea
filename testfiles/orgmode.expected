
hello org mode

# Normal org mode comment.
